modules/govuk_env_sync/files/govuk_env_sync.sh

```diff
@@ -682,6 +682,12 @@ nagios_code=2
 
 case ${action} in
   push)
+    if [ "${dbms}" == "mongo" ]; then
+      [ "$(is_writable_mongo)" != "true" ] \
+        && log "This machine is not a mongo master. Skipping." \
+        && trap - EXIT \
+        && exit
```

Comment:
It might be a bit easier to parse as:
```
if [ "${dbms}" == "mongo" ] && [ "$(is_writable_mongo)" != "true" ]; then
  log "This machine is not a mongo master. Skipping." \
  trap - EXIT
  exit
fi
```

Reply:
Absolutely agree - that was a hideous way of writing things. I've been doing a bunch of docker stuff, so one-lining things is beneficial.

I've made some changes - hopefully that's better now!

Reply:
> Absolutely agree - that was a hideous way of writing things. I've been doing a bunch of docker stuff, so one-lining things is beneficial.

No problem, exposure to Docker can be very harmful!

> I've made some changes - hopefully that's better now!

Yep, :+1: